WebAppForSolocoProject/Controllers/HomeController.cs

```diff
@@ -23,46 +26,274 @@ public HomeController(IConfiguration configuration, OwnerData ownerData)
+                {
+                    System.IO.File.Copy(file, Path.Combine(model.BasePath, model.SelectedOwnerName, model.PathToSaveFile, Path.GetFileName(file)), true);
+                    model.Logs.Add(file + " - file save in " + Path.Combine(model.BasePath, model.SelectedOwnerName, model.PathToSaveFile));
+                }
+            }
+            return View("Create", model);
+        }
+
+        private void BrowseFiles(TaskCompletionSource<List<string>> files)
+        {
+            var paths = new List<string>();
+
+            using (OpenFileDialog openFileDialog = new OpenFileDialog())
+            {
+                openFileDialog.Filter = Utility.importerFileExtensions;
+                openFileDialog.Multiselect = true;
+
+                if (openFileDialog.ShowDialog() == DialogResult.OK)
+                {
+                    foreach (string file in openFileDialog.FileNames)
+                    {
+                        paths.Add(file);
+                    }
+                }
+            }
+            files.SetResult(paths);
+        }
+
+        private async Task UpdateModel(HomeCreateVM model)
+        {
+            if (model.Logs == null)
+                model.Logs = new List<string>();
+            await Task.Run(() =>
+            {
+                model.OwnersList = ownerData.GetOwners();
+                model.SelectedOwner = model.OwnersList.FirstOrDefault(o => o.Name == model.SelectedOwnerName);
+                model.FolderList = model.SelectedOwner.QualityFolders;
+            });
+        }
+
+        public async Task<IActionResult> UpdateConfig(HomeCreateVM model)
+        {
+            await UpdateModel(model);
+
+            string config = ConfigurationManager.AppSettings["..."].ToString();
+            string[] configList = Utility.SplitCSL(@"\r?\n", config);
+
+            List<string> updateConfig = RewriteConfig(model, configList);
+
+            string path = await SaveToFile(updateConfig);
+
+            if (path != null)
+            {
+                model.Logs.Add("Config succesfully saved in "+ path);
+            }
+
+            return View("Create", model);
+        }
+        private List<string> RewriteConfig(HomeCreateVM model, string[] configList)
+        {
+            bool isOwnerConfig = false;
+            List<string> updateConfig = new List<string>();
+
+            foreach (var line in configList)
+            {
+                if (line.Contains("OwnersEnabled"))
+                {
+                    updateConfig.Add(". OwnersEnabled=" + model.SelectedOwnerName);
+                    continue;
+                }
+                if (line == ". " + model.SelectedOwnerName || line == ". " + model.SelectedOwner.ChildOwnerOf)
+                {
+                    isOwnerConfig = true;
+                }
+                if (line == string.Empty)
+                {
+                    isOwnerConfig = false;
+                }
```

Comment:
I don't like this source code lines with checking if line is equal to something . Move them to separate method and maybe use some regex.